frontend/src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {StyledDropdown, DropdownItem, Image} from "./Dropdown.styled";
+import { FaBell, FaFlag, FaEyeSlash } from "react-icons/fa";
+
+const Dropdown = () => {
+  const dropdownItems = [
+    {
+      icon: <FaBell />,
+      text: 'Notify me'
+    },
+    {
+      icon: <FaFlag />,
+      text: 'Report this post'
+    },
+    {
+      icon: <FaEyeSlash />,
+      text: "Don't show me this post"
```

Comment:
Raz ' a raz " 
Poprawnie skonfigurowany prettier powinien sam to naprawiać.

Reply:
tu jest don't przez co nie moglem dac '', chyba ze jest jakis inny sposob na to

Reply:
"Don\'t show me this post"

Reply:
uzyj \ przed '

Reply:
```suggestion
      text: 'Don\'t show me this post'
```

Reply:
ok dzięki za tipa !